src/Oro/Bundle/PimDataGridBundle/Resources/public/js/datagrid/column-selector.ts

```diff
@@ -476,6 +477,10 @@ class ColumnSelector extends BaseView {
       return;
     }
 
+    analytics.track('product-grid:column:selected', {
+      name: selected,
+    });
```

Comment:
The column selector is not only used in the product grid. You can add the grid name information in the tracked event

```suggestion
    analytics.track('product-grid:column:selected', {
      gridName: this.config.gridName,
      column: selected
    });
```